go/vt/vtadmin/cluster/cluster.go

```diff
@@ -1490,7 +1490,10 @@ func (c *Cluster) GetSchemas(ctx context.Context, opts GetSchemaOptions) ([]*vta
 
 		span.Annotate("cache_hit", ok)
 		if ok {
+			log.Infof("GetSchemas(cluster = %s) fetching schemas from schema cache", c.ID)
```

Comment:
This would be called every time the schemas page is loaded or an http/grpc call is made to `GetSchemas`. I can remove it if it's noisy! Just gives log clarity on whether or not the cache is being hit.